client/webserver/site/src/js/locales.ts

```diff
@@ -93,6 +93,7 @@ export const ID_MATCH_STATUS_REFUNDED = 'MATCH_STATUS_REFUNDED'
 export const ID_MATCH_STATUS_REFUND_PENDING = 'MATCH_STATUS_REFUND_PENDING'
 export const ID_MATCH_STATUS_REDEEM_PENDING = 'MATCH_STATUS_REDEEM_PENDING'
 export const ID_MATCH_STATUS_COMPLETE = 'MATCH_STATUS_COMPLETE'
+export const ID_TAKER_FOUND_MAKER_REDEMPTION = 'ID_TAKER_FOUND_MAKER_REDEMPTION'
```

Comment:
Remove the `ID_` prefix from the value.

Reply:
Hmm, you are sharp! Should be consistent with the rest now.